ui/app/src/components/shared/AssetList.vue

```diff
@@ -4,7 +4,7 @@
       <AssetItem class="token" :symbol="item.asset.symbol" />
       <div class="amount">{{ formatNumber(item.amount) }}</div>
       <div class="action">
-        <slot v-if="Number(item.amount.toFixed()) > 0" :asset="item"></slot>
+        <slot v-if="Number(item.amount.toFixed()) >= 0.00001" :asset="item"></slot>
```

Comment:
@iverc What should this number be?

Reply:
The reason I chose this value, is because that is the limits of decimals that we show to users. So if their balances are less than that we display 0.00000 anyway, so my assumption is that we should just hide the peg/unpeg if it looks like there balance is `0.00000` anyway.